app/models/resource.rb

```diff
@@ -8,4 +12,41 @@ class Resource < ActiveRecord::Base
   has_many :services
   has_many :ratings
   has_many :change_requests
+
+  if Rails.configuration.x.algolia.enabled
```

Comment:
@jjfreund had mentioned something about pulling this out into a module. Although, that might be even more confusing b/c algolia registers AR lifecycle events.